media-source-respec.html

```diff
@@ -622,11 +619,11 @@ <h3>Attributes</h3>
             <li>If the {{SourceBuffer/updating}} attribute equals true on any <a>SourceBuffer</a> in {{MediaSource/sourceBuffers}}, then throw an {{InvalidStateError}} exception and abort these steps.</li>
             <li>Run the [=end of stream=] algorithm with the error parameter set to |error:EndOfStreamError|.</li>
           </ol>
-        <table
-          class="parameters"><tbody><tr><th>Parameter</th><th>Type</th><th>Nullable</th><th>Optional</th><th>Description</th></tr><tr><th class="prmName">|error|</th><td class="prmType">{{EndOfStreamError}}</td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptTrue"><span role="img" aria-label="True">✔</span></td><td class="prmDesc"></td></tr></tbody></table><div><em>Return type: </em>{{undefined}}</div></dd>
-
           <dt><dfn><code>setLiveSeekableRange</code></dfn></dt><dd>
-
+          <p>
+            The |start| of the range, in seconds measured from [=presentation start time=]. While set, and if {{MediaSource/duration}} equals positive Infinity, {{HTMLMediaElement}}.{{HTMLMediaElement/seekable}} will return a non-empty TimeRanges object with a lowest range start timestamp no greater than |start|.
+            The |end| of range, in seconds measured from [=presentation start time=]. While set, and if {{MediaSource/duration}} equals positive Infinity, {{HTMLMediaElement}}.{{HTMLMediaElement/seekable}} will return a non-empty TimeRanges object with a highest range end timestamp no less than |end|.
+          </p>
           <p>Updates {{MediaSource/[[live seekable range]]}} that is used in
             <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a> to modify
             {{HTMLMediaElement}}.{{HTMLMediaElement/seekable}} behavior.</p>
```

Comment:
1. The now-removed redundant parameter tables were interestingly reordered in the rendered spec to occur between the summary of the method and the method-algorithm steps. With this method having non-trivial parameters that still need description, manual relocation needs to occur... (e.g. swap the order of these two paragraphs).
2. For readability, the non-trivial parameters might need a better introduction here.
3. The two "While set, and if ...." portions summarize the normative behavior defined in #htmlmediaelement-extensions section, so they are also redundant with that section. However, it might be good to retain some redundancy of that here. They shouldn't be normative here in the way they were, since they directly referenced the MediaSource's `duration`, which is not necessarily what the current version of the seekable extension does (it conditions reference on a "recent" duration and a "recent live seekable range" to handle MSE-in-Workers.

Also (not in scope of this PR): We might want to consider extracting the self-contained, but probably non-standard, text about conditionally cross-context-communicating worker's live seekable range updates and duration updates out of the seekable attribute's extension text. WDYT?

I haven't proofed this in respec, but maybe this would be a good fix for now:

```suggestion
          <p>Updates {{MediaSource/[[live seekable range]]}} that is used in
            <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a> to modify
            {{HTMLMediaElement}}.{{HTMLMediaElement/seekable}} behavior.</p>
          <p>
            The |start| of the range, in seconds measured from [=presentation start time=].
            The |end| of range, in seconds measured from [=presentation start time=].
          </p>
          <p class="note">
            <a href="#htmlmediaelement-extensions">HTMLMediaElement Extensions</a> uses
            {{MediaSource/[[live seekable range]]}}, if not empty and if
            {{MediaSource/duration}} is positive infinity, to return a {{TimeRanges}}
            object with a lowest range start timestamp no greater than |start| and a highest
            range end timestamp no less than |end|. If this is a worker {{MediaSource}},
            cross-context communication latency will delay the effect of any duration or
            live seekable range updates on
            {{HTMLMediaElement}}.{{HTMLMediaElement/seekable}} behavior.
          </p>
         
```

Reply:
> Also (not in scope of this PR): We might want to consider extracting the self-contained, but probably non-standard, text about conditionally cross-context-communicating worker's live seekable range updates and duration updates out of the seekable attribute's extension text. WDYT?

Sounds good. Yes.